src/shapes/text.class.js

```diff
@@ -348,6 +348,9 @@
     enlargeSpaces: function() {
       var diffSpace, currentLineWidth, numberOfSpaces, accumulatedSpace, line, charBound, spaces;
       for (var i = 0, len = this._textLines.length; i < len; i++) {
+        if (i === len - 1 || this.isEndOfWrapping(i)) {
```

Comment:
ok here needs a this.textAlign !== 'justify' && ( ..... )

Reply:
i still think this change is needed.
Since we are giving the user

justify-left, jsutify-right, justify-center,  this thing here should change that we do not want the old plain `justify`.

Who built an app over 2.0b7 or earlier, does not want to see old JSON saved with plain justify loaded differently just because we think is better to have last line not justified.

In this way we give an option to everyone at expense of littled added complexity, and we do not break old stuff.

Reply:
yes, i agree. backwards compatibility is important